cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -385,6 +388,17 @@ You cannot use the following reserved CIDR blocks:
 169.254.169.252/30
 ----
 
+You can configure the following CIDRS from the MuleSoft infrastructure CIDRs pool as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
+
+----
+100.64.0.0/16
+100.66.0.0/16
+100.67.0.0/16
+100.68.0.0/16
+----
+
+Additionally, AWS EKS uses `10.100.0.0/16` or `172.20.0.0/16` inside of private space networks. 
```

Comment:
let's come back to this later after reserved CIDR is documented.

Reply:
```suggestion
// Additionally, AWS EKS uses `10.100.0.0/16` or `172.20.0.0/16` inside of private space networks. 
```